### ISSUE TITLE
Missing lang attribute on <html>

### ISSUE THREAD

**Issue by coleweusthem** [asc-csa/AlouetteApp#18]
\<html\> element must have a lang attribute.
<img width="582" alt="Screen Shot 2021-03-10 at 1 53 14 PM" src="https://user-images.githubusercontent.com/46227765/110674129-f7b09b00-81a7-11eb-9fd3-15c816069842.png">

**Comment by coleweusthem** [comment]
html now has a lang attr
